cmd/driver/config/config.go

```diff
@@ -0,0 +1,219 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2023 The Falco Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package driverconfig
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/pterm/pterm"
+	"github.com/spf13/cobra"
+	"golang.org/x/net/context"
+	v1 "k8s.io/api/core/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"github.com/falcosecurity/falcoctl/internal/config"
+	"github.com/falcosecurity/falcoctl/internal/utils"
+	driverdistro "github.com/falcosecurity/falcoctl/pkg/driver/distro"
+	driverkernel "github.com/falcosecurity/falcoctl/pkg/driver/kernel"
+	drivertype "github.com/falcosecurity/falcoctl/pkg/driver/type"
+	"github.com/falcosecurity/falcoctl/pkg/options"
+)
+
+type driverConfigOptions struct {
+	*options.Common
+	Type     string
+	Version  string
+	Repos    []string
+	Name     string
+	HostRoot string
+	Update   bool
+}
+
+// NewDriverConfigCmd configures a driver and stores it in config.
+func NewDriverConfigCmd(_ context.Context, opt *options.Common) *cobra.Command {
+	o := driverConfigOptions{
+		Common: opt,
+	}
+
+	cmd := &cobra.Command{
+		Use:                   "config [flags]",
+		DisableFlagsInUseLine: true,
+		Short:                 "Configure a driver",
+		Long:                  "Configure a driver for future usages with other driver subcommands",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return o.RunDriverConfig(cmd)
+		},
+	}
+
+	cmd.Flags().StringVar(&o.Type, "type", "", "Driver type to be configured. Allowed: "+strings.Join(drivertype.GetDriverTypes(), ",")+",auto")
+	cmd.Flags().StringVar(&o.Version, "version", "", "Driver version to be configured.")
+	cmd.Flags().StringSliceVar(&o.Repos, "repo", nil, "Driver repo to be configured.")
+	cmd.Flags().StringVar(&o.Name, "name", "", "Driver name to be configured.")
+	cmd.Flags().StringVar(&o.HostRoot, "host-root", "", "Driver host root to be configured.")
```

Comment:
We should be able to set defaults for `type`, `repo`, and `name`. 
We can't set a default for `host-root` (its value usually changes depending on the context). Also, we should not set a default for `version`, since it may change at any time. 

Does it make sense?